backend/python/app/services/implementations/language_service.py

```diff
@@ -26,3 +26,12 @@ def add_language(self, language, is_rtl):
         except Exception as error:
             self.logger.error(str(error))
             raise error
+
+    def get_is_rtl(self, language):
+        try:
+            to_check = Language.query.filter_by(language=language).first()
+
+            return {"is_rtl": to_check.is_rtl}
+        except Exception as error:
```

Comment:
i know you have this except but can you add an extra err catch in the try if the language does not exist in the db? eg
![image](https://user-images.githubusercontent.com/15113249/148485295-ae333f19-842f-45a5-bcd0-24a707eba971.png)
i just feel like querying the wrong language will happen often enough that a special message for it would be helpful